swan-lake/development-tutorials/source-code-dependencies/style-guide/annotations-documentation-and-comments.md

```diff
@@ -42,55 +42,28 @@ function testFunction() {
   
 ```ballerina
 // Parameter annotation.
-public function secureFunction1(@untainted string secureInName, @untainted int secureInId, string insecureIn) {
+public function secureFunction1(string secureInName, int secureInId, string insecureIn) {
```

Comment:
Since we are removing tainted/untainted, we'll have to use some other annotation since this specific point talks about parameter annotations

Reply:
Updated.